unified-installer/astra-unified-installer.sh

```diff
@@ -1735,22 +1798,48 @@ step_check_volumesnapshotclasses() {
     fi
 }
 
-step_check_namespace_exists() {
-    if [ -z "$(kubectl get namespace "$NAMESPACE" -o name 2> /dev/null)" ]; then
-        if [ -n "$IMAGE_PULL_SECRET" ]; then
-            local err_msg="The specified namespace '$NAMESPACE' does not exist on the cluster, but IMAGE_PULL_SECRET is set."
-            err_msg+=" Please create the namespace and secret, or unset IMAGE_PULL_SECRET."
-            add_problem "namespace '$NAMESPACE': not found but IMAGE_PULL_SECRET is set" "$err_msg"
+step_check_namespace_and_pull_secret_exist() {
```

Comment:
The main changes in this function are adding pre-constructed commands to the error message to make it easier for the user to correct issues related to a missing namespace/pull secret. For example:
```
--- Cluster management failed! We've identified the following issues:
* The specified NAMESPACE 'astra' does not exist on the cluster, but IMAGE_PULL_SECRET is set. Please create the namespace and secret:
    - kubectl create namespace 'astra'
    - kubectl create secret docker-registry 'test-pull-secret' -n 'astra' --docker-server='docker.repo.eng.netapp.com' --docker-username='<USERNAME>' --docker-password='<PASSWORD>'
```